common/defs/bitcoin/zcoin.json

```diff
@@ -4,7 +4,7 @@
   "coin_label": "Zcoin",
   "website": "https://zcoin.io",
   "github": "https://github.com/zcoinofficial/zcoin",
-  "maintainer": "Yura Pakhuchiy <pakhuchiy@gmail.com>",
+  "maintainer": "Zcoin Team <team@zcoin.io>",
```

Comment:
Please add your personal email if you want to maintain Zcoin. Adding "team" email is not helping anyone.

Reply:
Done.